content/browser/service_manager/service_manager_context.cc

```diff
@@ -671,6 +675,24 @@ ServiceManagerContext::ServiceManagerContext(
         base::BindRepeating(&base::ASCIIToUTF16, "Visuals Service");
   }
 
+#if defined(CASTANETS)
+  // TODO(is46.kim) : All out of process services except for NetworkService
+  // should be launched in browser process side, not remote process.
+  if (!base::CommandLine::ForCurrentProcess()->HasSwitch(
+          switches::kEnableForking)) {
+    for (auto it = out_of_process_services.cbegin();
+         it != out_of_process_services.cend();) {
+      if (network_service_enabled && !network_service_in_process &&
+          it->first == mojom::kNetworkServiceName) {
+        it = std::next(it);
+      } else {
+        LOG(INFO) << "Disable out-of-process service : " << it->first;
+        it = out_of_process_services.erase(it);
```

Comment:
Can we 'break' here?

Reply:
it needs to continue to remove all services.

Reply:
My mistake. I thought we only remove network service from the list.